src/partials/redpanda-connect-content.hbs

```diff
@@ -0,0 +1,106 @@
+                <button class="redpanda-connect-tab-button">Docker</button>
+                <button class="redpanda-connect-tab-button">Asdf</button>
+            </div>
+            <div class="redpanda-connect-code-snippet">
+                <pre><code>
+# Install
+curl -Lsf https://www.benthos.dev/sh/install | bash
+
+# Make a config
+benthos create nats/protobuf/aws_sqs > ./config.yaml
+
+# Run
+benthos -c ./config.yaml
+                </code></pre>
+            </div>
+          </div>
+
+          <div class="redpanda-connect-feature">
+            <div class="redpanda-connect-config-example">
+                <div class="redpanda-connect-tabs">
+                    <button class="redpanda-connect-tab-button active">Mapping</button>
+                    <button class="redpanda-connect-tab-button">Multiplexing</button>
+                    <button class="redpanda-connect-tab-button">Windowing</button>
+                    <button class="redpanda-connect-tab-button">Enrichments</button>
+                </div>
+                <pre><code>
+input:
+  gcp_pubsub:
+    project: foo
+    subscription: bar
+
+pipeline:
+  processors:
+    - mapping: |
+        root.message = this
+        root.meta.link_count = this.links.length()
+        root.user.age = this.user.age.number()
+
+output:
+  redis_streams:
+    url: tcp://TODO:6379
+    stream: bar
+    max_in_flight: 20
+                </code></pre>
+                <button class="redpanda-connect-read-more">Learn more &rarr;</button>
+            </div>
+          </div>
+
+          <div class="redpanda-connect-feature">
+            <h3 class="redpanda-connect-feature-title">Efficiently Handles Routine Tasks</h3>
+            <p class="redpanda-connect-feature-description">Redpanda Connect addresses common data engineering challenges such as transformations, integrations, and multiplexing through declarative and unit-testable configurations. Those features allow seamless and incremental adjustments to data pipelines as requirements evolve, so you can focus on your product.</p>
+          </div>
+
+          <div class="redpanda-connect-feature">
+            <h3 class="redpanda-connect-feature-title">Highly Integrated</h3>
+            <p class="redpanda-connect-feature-description">Redpanda Connect can integrate a wide array of sources and sinks and connect to various databases, caches, HTTP APIs, lambdas, and more, facilitating seamless integration with your existing infrastructure.</p>
+            <p class="redpanda-connect-feature-description">Working with disparate APIs and services can be a daunting task, especially in a streaming data context. With Redpanda Connect, it is possible to decompose these tasks and automatically parallelize them as part of a streaming workflow.</p>
+          </div>
+
+          <div class="redpanda-connect-video-container">
+            <iframe src="https://www.youtube.com/embed/uvbp2LCmQMY" frameborder="0" allowfullscreen></iframe>
```

Comment:
Ca you please replace this link with: https://youtu.be/DYEphcjofgE

Reply:
The original video is about benthos studio tho. They're not equivalent.
Should I add the above anyway ?